align_system/prompt_engineering/kdma_descriptions.yml

```diff
@@ -1,35 +1,52 @@
 MoralDesert:
   name: moral deservingness
-  description: Moral deservingness refers to rewarding moral actions and punishing immoral misdeeds, which is related to concepts of justice. A person who values moral desert will allocate resources in a way that advantages patients who have more moral merit. Low moral deserts consideration is when a casualty’s blame or responsibility for the situation is not considered. A person with high moral deserts consideration will favor those who are victims, bystanders, or helpers/responders over those who are perpetrators or otherwise potentially blame-worthy. A person with low moral deserts consideration will not take a casualty’s degree of responsibility for the situation into account when making resource allocation decisions.
-  score_examples: Consider a situation where patient A and patient B are both injured. If both patients acted in a morally neutral manner, choosing to treat patient A would receive a score of 5 for moral deservingness. However if patient A caused harm or acted with poor morality, choosing to treat patient A would receive a score of 0, while choosing to treat patient B would receive a score of 10.
-  valid_scores: [0,1,2,3,4,5,6,7,8,9,10]
+  description: Moral deservingness refers to rewarding moral actions and punishing immoral misdeeds, which is related to concepts of justice. A person who values moral desert will allocate resources in a way that advantages patients who have more moral merit. Low moral deserts consideration is when a casualty's blame or responsibility for the situation is not considered. A person with high moral deserts consideration will favor those who are victims, bystanders, or helpers/responders over those who are perpetrators or otherwise potentially blame-worthy. A person with low moral deserts consideration will not take a casualty's degree of responsibility for the situation into account when making resource allocation decisions.
+  factor: 10
+  score_examples: 'Consider a situation where patient A and patient B are both injured. If both patients acted in a morally neutral manner, choosing to treat patient A would receive a score of {{ "%g" | format(0.5 * kdma_scale_factor) }} for moral deservingness. However if patient A caused harm or acted with poor morality, choosing to treat patient A would receive a score of {{ "%g" | format(0 * kdma_scale_factor) }}, while choosing to treat patient B would receive a score of {{ "%g" | format(1.0 * kdma_scale_factor) }}.'
```

Comment:
I'm assuming the `"%g"` here formats into an integer?  (It's tough to google such a thing)

Reply:
It's "standard number format" or something along those lines, but yes 10.0 becomes 10, but 10.5 would stay 10.5